docs/test-plans/001-Big-Blocks/test-cases/tc-001-val-large-txs.md

```diff
@@ -0,0 +1,28 @@
+# Test-Case #001 - Validators submit large txs
+
+## Pre-Requisites:
+
+1. Every validator has enough funds in the account
+2. The chain has created the first block
+3. Every validator has enough peers
+4. Validators’ set is not changing during test execution
+   1. All validators are created during genesis
+
+## Steps for each of the validators:
+
+1. Setups network with:
+   1. `I` mb of bandwidth
+   2. `J` milliseconds of network latency
+2. Generates and broadcasts
+   1. `X` kb of random data
+   2. `Y` times
+3. Checks the block size is bigger then 3.5 MiB
```

Comment:
Why 3.5?

Reply:
We ideally want 4MB so I guess 3.5 is a good `>` metric ?

Reply:
1. Most of the times we can't get higher then 4mil bytes, which is around 3.8 mb
2. I want to leave some room to not the fail the test-case because we are lower then 3.8Mb